app/src/main/java/com/keylesspalace/tusky/MainActivity.java

```diff
@@ -15,22 +15,22 @@
 
 package com.keylesspalace.tusky;
 
-import android.arch.lifecycle.Lifecycle;
+import androidx.lifecycle.Lifecycle;
 import android.content.Intent;
 import android.graphics.Color;
 import android.graphics.drawable.Drawable;
 import android.net.Uri;
 import android.os.Bundle;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+import com.google.android.material.tabs.TabLayout;
+import androidx.emoji.text.EmojiCompat;
+import androidx.fragment.app.Fragment;
+import androidx.core.content.ContextCompat;
+import androidx.viewpager.widget.ViewPager;
+import androidx.appcompat.app.AlertDialog;
 import android.os.Handler;
```

Comment:
Studio complains about `supportFragmentInjector`. If it's something real, we should check other places

Reply:
It somehow wants `android.support.v4.app.Fragment;`

Reply:
It fails to build for me because of that :thinking:

Reply:
Did you clean build cache? I get that as well everytime I switch branch